README.md

```diff
@@ -82,7 +85,47 @@ La app aún está en una etapa muy temprana del desarrollo por lo que podrían h
+- _API_ → `backend/.env.staging`
+- _servidor web_ → `frontend/.env.staging`
+- _base de datos_ → `database/.env.staging`
+
+Luego, para correr la aplicación utilizando un servidor mock de **CAS externo** se debe:
+1. Definir las variables `CAS_SERVER_URL` y `CAS_LOGIN_REDIRECTION_URL` en `backend/.env` con la URL del servidor externo.
+2. Levantar los contenedores con `docker compose up -d --build` desde la raíz del repositorio.
+3. Finalmente, se puede detener la app con `docker compose down` desde la misma ubicación.
+
+Alternativamente, para correr la aplicación utilizando un servidor mock de **CAS local**:
+1. Dejar las variables `CAS_SERVER_URL` y `CAS_LOGIN_REDIRECTION_URL` en `backend/.env` con los valores predeterminados del archivo de ejemplo `.env.staging`.
+2. Luego, es necesario generar el archivo `cas-mock-users.json` en `cas-mock/data` a partir del ejemplo `cas-mock-users.json.example`.
+3. Levantar los contenedores con `docker compose -f docker-compose.yml -f docker-compose.cas-mock.yml up -d --force-recreate --build` desde la raíz del repositorio.
+4. Finalmente, se puede detener la app con `docker compose -f docker-compose.yml -f docker-compose.cas-mock.yml down` desde la misma ubicación.
+
+### Producción
+
+El ambiente de producción es manejado por la universidad de forma interna, por lo que aquí se detallan las **instrucciones para desplegar el planner** de forma manual:
+1. Se deben crear tres archivos `.env`, uno por cada servicio y dentro de su respectiva carpeta:
+- `backend/.env` a partir del ejemplo `backend/.env.production` (_API_)
+- `frontend/.env` a partir del ejemplo `frontend/.env.production` (_servidor web_).
+- `database/.env` a partir del ejemplo `database/.env.production` (_base de datos_).
+2. Reemplazar los valores de las variables de entorno según corresponda en todos los archivos `.env` creados. **IMPORTANTE:** no olvidar modificar la variable `JWT_SECRET` en `backend/.env` y otras variables que puedan contener secretos para evitar vulnerabilidades de seguridad.
+3. Levantar los contenedores con `docker compose up -d --build` desde la raíz del repositorio. Requiere _Docker_ y _Docker Compose_ instalados en la máquina.
+4. Revisar el estado de los contenedores con `docker ps` o `docker container ls`.
+5. Finalmente, se puede detener la app con `docker compose down` desde la misma ubicación.
+
+Nota: los comandos podrían variar ligeramente dependiendo del sistema operativo y versión de *Docker Compose*. En particular, podría ser necesario utilizar `docker-compose` en vez de `docker compose` y `sudo docker compose` en vez de `docker compose`.
+
+---
+
+Cabe mencionar que sería ideal a futuro implementar un **despliegue automático del planner** utilizando técnicas de *CI/CD*, pero de momento esta opción se pospone debido a las restricciones de seguridad y requerimientos de la universidad para hacer un despliegue interno.
```

Comment:
Mi hot take es que debiésemos hacer esto a staging (algo que es súper low effort si usamos algo como push to deploy).

Reply:
Pero si staging está en los servidores de la U será complejo igual, no? O te refieres a un staging nuestro que tengamos afuera (como el que ya levantamos de prueba)?

Reply:
No es nada dificil con Tailscale: https://tailscale.com/blog/2021-05-github-actions-and-tailscale/

Reply:
(Esto es para otra PR en todo caso)